recipes/sklearn-quantile/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "sklearn-quantile" %}
+{% set version = "0.0.18" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sklearn_quantile-{{ version }}.tar.gz
+  sha256: c3734f9bcbc86cadf0bb255deb87b7d4a0d24a201f452bd26d6ebe756e665b18
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython
+    - numpy
+    - pip
+    - python
+    - setuptools
+  run:
+    - python
+    - scikit-learn >=1.0
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - sklearn_quantile
+    - sklearn_quantile.utils
+    - sklearn_quantile.ensemble
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jasperroebroek/sklearn-quantile
+  license: BSD-3-Clause
+  license_file: LICENSE
+  summary: 'Scikit-learn compatible quantile machine learning methods'
+  doc_url: https://sklearn-quantile.readthedocs.io/en/latest/?badge=latest
+  dev_url: https://github.com/jasperroebroek/sklearn-quantile
+
+extra:
+  recipe-maintainers:
+    - jroebroek
```

Comment:
This is a different username than the one you are using for the PR. Intended?

Reply:
That was not intentional, thanks for spotting. I have committed the change.